scripts/test-example-kafka.json

```diff
@@ -4,8 +4,9 @@
   "msgsPerSecond": 100000,
   "msgSizeBytes": 1024,
   "batchSizeSend": 1000,
+  "senderConcurrency": 200,
   "batchSizeReceive": 1000,
-  "maxSendInFlight": 10000,
+  "receiverConcurrency": 200,
```

Comment:
sender is ok being 200, but for receiver this might be too high - a couple of processes running the receive loop should be enough (there's one connection per kafka consumer anyway)

Reply:
Changed to 8